generator/osm2type.cpp

```diff
@@ -1478,6 +1478,10 @@ void GetNameAndType(OsmElement * p, FeatureBuilderParams & params,
       {
         params.SetDefaultNameIfEmpty(std::move(v));
       }},
+      {"is_sidepath:of:name", "*", [&params](string & k, string & v)
```

Comment:
This block was designed for pistes according to the comment and implementation, right?
  // Fetch piste:name and piste:ref if there are no other name/ref values.

This code tries to set the name of the sidepath from the main street, right? Why?

Reply:
> This block was designed for pistes according to the comment and implementation, right?

oops, moved to main block

> This code tries to set the name of the sidepath from the main street, right? Why?

No, it just sets the `name` to the value of the tag `is_sidepath:of:name` when `name` isn't already tagged. just a simple tag substitution like those in replaced_tags.txt :)